### ISSUE TITLE
Metric of hyphen, hyphen-minus, etc

### ISSUE THREAD

**Issue by stone-zeng** [bBoxType/FiraGO#26]
The metric of hyphen, hyphen-minus, etc really look strange:

![image](https://user-images.githubusercontent.com/18233088/50004985-be4a4e80-ffe3-11e8-8412-40786bae81d2.png)

![image](https://user-images.githubusercontent.com/18233088/50005311-ec7c5e00-ffe4-11e8-975d-fbd58dec47ab.png)

This issue is related to `U+002D`, `U+00AD`, `U+2010` and `U+2011`.

See <https://github.com/firamath/firamath/issues/24>.

**Comment by carrois** [comment]
Yes, should be centered. We’ll include it in the next update. Thank you!
